bin/check_pylint_diff

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Copyright (C) 2016 Google Inc., authors, and contributors <see AUTHORS file>
+# Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+# Created By: miha@reciprocitylabs.com
+# Maintained By: miha@reciprocitylabs.com
+
+set -o nounset
+set -o errexit
+
+ARG1=${1:-}
+TMP_REPO=".temp_repo"
+SCRIPT=$(basename "$0")
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+
+cd "${SCRIPTPATH}/.."
+
+
+if [[ "$ARG1" == "-h" || "$ARG1" == "--help" ]]; then
+  echo "
+Usage: $SCRIPT [test_commit] [-h]
+
+This script will compare pylint error count from two different commits.
+Note: all changes that are not committed will be ignored.
+
+
+Given the commit tree:
+
+       D---E---F---G---H
+            \\         /
+             A---B---C
+
+- Running '$SCRIPT' on C will check the diff between B and C
```

Comment:
> only changes in the last commit. last commit is basically the diff between previous commit and current state

Yes, and in case of pull requests, that's not enough since PRs usually consist of multiple commits?

The use case I see is this: We have this software. We want to ensure no code from PRs is merged into master if it introduces more lint errors than it fixes.

Does this PR, as-is, satisfy that use case?